content/docs/introduction/install.md

```diff
@@ -85,3 +85,20 @@ docker run -p 9090:9090 my-prometheus
 
 A more advanced option is to render the config dynamically on start
 with some tooling or even have a daemon update it periodically.
+
+## Using Configuration Management Systems
```

Comment:
We don't use Title Case in docs headlines, so "Using configuration management systems".